tests/relay_integration/lang/javascript/test_plugin.py

```diff
@@ -176,12 +176,23 @@ def test_adds_contexts_with_device(self):
             "type": "os",
             "version": "4.3",
         }
-        assert contexts.get("browser") == {
+
+        browser_context = contexts.get("browser")
+
+        # The `user_agent` field was added retroactively so the browser context assertion needs to be forwards and backwards compatible
+        assert (
+            browser_context.get("user_agent") == None
+            or browser_context.get("user_agent")
+            == "Mozilla/5.0 (Linux; U; Android 4.3; en-us; SCH-R530U Build/JSS15J) AppleWebKit/534.30 (KHTML, like Gecko) Version/4.0 Mobile Safari/534.30 USCC-R530U"
+        )
+        del browser_context.user_agent
```

Comment:
This is just a `browser_context.pop("user_agent", None) in (None, "...")`

Reply:
my python skillz are showing